### DISCUSSION TITLE
How do I create Github Pages using Nuxt3?

### DISCUSSION THREAD

**Discussion by eesoymilk** [nuxt/nuxt#16989] [Questions]
I've tried various ways to push my static site up using `yarn generate` command and `ssr: false` in the config.
But I keep on getting error saying that `.mjs` is a disallowed MIME type when I open up my github.oi page.
I tried compiling all `.mjs` files to `.js` ones but still got the same error.

This is the error I got in the browser's console.
```
Loading module from “https://<mygithubusername>.github.io/_nuxt/entry.<hash>.mjs” was blocked because of a disallowed MIME type (“text/html”).
```

**Comment by danielroe** [comment]
On edge channel we now output `.js` files. https://v3.nuxtjs.org/guide/going-further/edge-channel/
